pkg/tenant/db/members.go

```diff
@@ -100,10 +99,10 @@ func ListMembers(ctx context.Context, tenantID ulid.ULID) (members []*Member, er
 	members = make([]*Member, 0, len(values))
 	for _, data := range values {
 		member := &Member{}
-		if err = member.UnmarshalValue(data); err != nil {
-			fmt.Println(err)
-			return nil, err
-		}
+
+		// Marshal and unmarshal the data with msgPack.
+		member.MarshalPackData()
```

Comment:
The list handlers were returning empty arrays because the data ( a `[]byte`) needs to be marshaled and unmarshaled by msgPack. Previously, it was only unmarshaled. A method was created to make this a little cleaner.